demo/dlb-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+ENABLED_DLB_PF_PCIIDS=${ENABLED_DLB_PF_PCIIDS:-270b 2710 2714}
+NUMVFS=${NUMVFS:-4}
+
+bind() {
+    BSF=$1
+    DRIVER=$2
+
+    LOOP=0
+    while [ "$LOOP" -lt  5 ]
```

Comment:
I'd prefer for loops like:
```
for i in $(seq 5)
do

done
```
So you don't have to remember to add the +1 in some new code branch later.

And maybe store the magic 5 in a variable in the beginning of the file.

Reply:
it's just copied and pasted from qat thing you gave..!
Let me change like that.! :)